### DISCUSSION TITLE
Pressing esc/back button after watching replay in playlist should take you back to the playlist

### DISCUSSION THREAD

**Discussion by sz-da** [ppy/osu#26613] [Ideas] [answered]
It currently takes you to the main menu, which is a bit confusing imo

https://github.com/ppy/osu/assets/76718358/d6288e6d-ae8b-491a-a693-8891edc323e9

**Comment by peppy** [comment]
I agree. This will require a change to how `PresentScore` works.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/issues/26666
